7forecast.tex

```diff
@@ -177,7 +177,7 @@ \subsection{Forecast File Options (forecast.ss)}
 
   %\hline
   \pagebreak
-  1 \Tstrut & \hyperlink{ForeSpawn}{Base recruitment in forecast:} & \multirow{1}{1cm}[-0.25cm]{\parbox{11cm}{This option controls the base R (to which devs are applied) in the forecast, or the averaging of a range of historical Rs after all adjustments and devs were applied. For options 1 and 2, the next value read is a scalar applied to the base. Option 3 also averages the distribution of R among areas and morphs, but this is now redundant with the Control Averaging input two boxes down. Options 3 and 4 require that the user set the forecast R\_dev phase to negative and that the last year of rec\_devs is the end year.}} \\
+  1 \Tstrut & \hyperlink{ForeSpawn}{Base recruitment in forecast:} & \multirow{1}{1cm}[-0.25cm]{\parbox{11cm}{This option controls the base R (to which deviations are applied) in the forecast, or the averaging of a range of historical Rs after all adjustments and deviations were applied. For options 1 and 2, the next value read is a scalar applied to the base. Option 3 also averages the distribution of R among areas and morphs, but this is now redundant with the Control Averaging input two boxes down. Options 3 and 4 require that the user set the \hyperlink{FcastRecDevPhase}{forecast recruitment deviation phase} to negative and that the \hyperlink{RecDevEndYear}{last year of recruitment deviations} is the \hyperlink{EndYear}{end year}.}} \\
```

Comment:
In order to clean up the language we should change "R" to recruitment or recruitments throughout to be clear. Additionally, I requested that Rick make the output text by SS3 in the forecast to be consistent between using average or mean.  I believe that the language was changed to "mean" and this text should be updated accordingly.